src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -766,6 +776,9 @@ u16 EnGo2_GetTextId(PlayState* play, Actor* thisx) {
                 return EnGo2_GetTextIdGoronMarketBazaar(play, this);
         }
     }
+#ifdef AVOID_UB
+    return faceReaction; // always 0 at this point
```

Comment:
imo this could use a comment like "at this point in vanilla mq debug v0 is the return from Text_GetFaceReaction"